XLS-0075d-account-permission-delegation

```diff
@@ -0,0 +1,309 @@
+<pre>
```

Comment:
Move this file under the specified directory and the file extension should be set correctly.